charts/operator/Chart.yaml

```diff
@@ -5,10 +5,15 @@ icon: https://raw.githubusercontent.com/rancher/elemental/main/logo/icon-element
 version: 0.0.0
 appVersion: 0.0.0
 annotations:
+  catalog.cattle.io/auto-install: elemental-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Elemental
-  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.28.0-0'
   catalog.cattle.io/namespace: cattle-elemental-system
   catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
   catalog.cattle.io/provides-gvr: elemental.cattle.io/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
```

Comment:
Same here, this also seem rather strict

Reply:
Well, this is different IMO: we don't support Rancher 2.6.x.
Moreover, we don't know if this chart will be compatible with Rancher 2.8.0: a lot could change there.
We will expand the supported versions when needed, but for sure we want the older chart versions to disappear by default from the marketplace for future Rancher versions (we want to release newer charts... or rebuild the same with updated tags is needed after proper testing).
This also is a common pattern in Rancher Marketplace charts.